Objective-C/Internal/CBLData.mm

```diff
@@ -38,7 +38,7 @@ - (id) cbl_toCBLObject {
     if (self != kCBLRemovedValue) {
         [NSException raise: NSInternalInconsistencyException
                     format: kCBLErrorMessageInvalidCouchbaseObjType, [self class],
-         @"NSNumber, NSString, NSDate, CBLDictionary, CBLArray"];
+         @"NSArray, NSDictionary, NSDate, NSString, NSNumber, NSNull, CBLArray, CBLMutableArray, CBLDictionary, CBLMutableDictionary, CBLNewDictionary, CBLBlob"];
```

Comment:
I got what you mean now about `CBLNewDictionary`. CBLNewDictionary is an internal object so it shouldn't be in the list.